modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -646,7 +646,15 @@ def to_pandas(cls, partitions):
             isinstance(obj, (pandas.DataFrame, pandas.Series))
             for obj in retrieved_objects
         ):
-            retrieved_objects = np.reshape(retrieved_objects, partitions.shape)
+            height, width = 0, 0
+            if len(partitions.shape) == 2:
+                # possible case when shape == (0,)
+                height, width = partitions.shape
```

Comment:
```suggestion
            height, width, *_ = tuple(partitions.shape) + (0,)
```
utilizing [PEP3132](https://peps.python.org/pep-3132/)